2013/01/09 13:33:40 [error] [php] Object of class CInlineAction could not be converted to string (C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php:9)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): YumRestController->runActionWithFilters()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): YumRestController->run()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/users
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php (9)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/09 13:35:47 [error] [php] Undefined variable: mode (C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php:125)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): YumRestController->runAction()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): YumRestController->runActionWithFilters()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): YumRestController->run()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/users
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php (125)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/09 13:36:48 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:336
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('view')
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('user/rest/view')
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/api/users/1
---
2013/01/09 13:36:59 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:336
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('view')
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('user/rest/view')
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/api/users/1
---
2013/01/09 14:02:07 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php:237)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): YumRestController->runAction()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): YumRestController->runActionWithFilters()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): YumRestController->run()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php (237)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php (13)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/09 14:03:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:336
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('view')
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('user/rest/view')
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/api/user/1
---
2013/01/09 14:04:14 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:336
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('view')
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('user/rest/view')
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/api/user/1
---
2013/01/09 14:12:29 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php:217)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php (217)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php (29)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/09 14:13:55 [error] [php] include(YumRestController.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\Users\Arsey\Dropbox\sites\YII_1_1_13\YiiBase.php:421)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php(29): spl_autoload_call()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): ApiController->actionList()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php (29)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/09 14:15:36 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.YumRestController" is not defined.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CComponent.php:130
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CModule.php(105): CComponent->__get('YumRestControll...')
#1 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php(30): CModule->__get('YumRestControll...')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): ApiController->actionList()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/api/feedbacks
---
2013/01/09 14:15:50 [error] [exception.CException] exception 'CException' with message 'Property "CWebApplication.YumRestController" is not defined.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CComponent.php:130
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CModule.php(105): CComponent->__get('YumRestControll...')
#1 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php(30): CModule->__get('YumRestControll...')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): ApiController->actionList()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/api/feedbacks
---
2013/01/09 14:17:20 [error] [php] Missing argument 1 for CController::__construct(), called in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php on line 29 and defined (C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:110)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php (29)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/09 14:18:31 [error] [exception.CException] exception 'CException' with message 'YumRestController and its behaviors do not have a method or closure named "_checkAuth".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CComponent.php:265
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php(30): CComponent->__call('_checkAuth', Array)
#1 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\controllers\ApiController.php(30): YumRestController->_checkAuth()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): ApiController->actionList()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/api/feedbacks
---
2013/01/10 14:01:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:01:32 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php:235)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): YumRestController->runAction()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): YumRestController->runActionWithFilters()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): YumRestController->run()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CWebApplication->run()
REQUEST_URI=/user/rest/list
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php (235)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\protected\modules\user\controllers\YumRestController.php (11)
in C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php (13)
2013/01/10 14:01:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:01:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:01:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:03:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:03:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:03:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:04:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:06:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:06:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:06:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:06:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/10 14:34:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "feedbacks".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('feedbacks')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('feedbacks')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/feedbacks/p...')
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/feedbacks/published
---
2013/01/10 14:34:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "feedbacks".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('feedbacks')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('feedbacks')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/feedbacks/p...')
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/feedbacks/published
---
2013/01/10 14:34:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "feedbacks".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('feedbacks')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('feedbacks')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/feedbacks/p...')
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/feedbacks/published
---
2013/01/10 14:37:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "feedbacks".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('feedbacks')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('feedbacks')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/feedbacks/p...')
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/feedbacks/published
---
2013/01/10 14:37:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "feedbacks".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('feedbacks')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('feedbacks')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/feedbacks/p...')
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\Arsey\Dropbox\sites\PLANTEATERS\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/feedbacks/published
---
2013/01/11 11:09:11 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php:218)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (218)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 11:09:22 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php:218)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (218)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 11:10:12 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php:218)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (218)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 11:10:14 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php:218)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (218)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 11:10:14 [error] [php] Undefined variable: username (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php:218)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (218)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 11:10:19 [error] [php] Undefined variable: password (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php:218)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): ApiController->runAction()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): ApiController->runActionWithFilters()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): ApiController->run()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/api/feedbacks/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (218)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\ApiController.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 11:54:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 11:54:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 11:54:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 11:54:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 11:55:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:54:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:56:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:58:10 [error] [exception.CException] exception 'CException' with message 'CWebApplication and its behaviors do not have a method or closure named "googlePlaces".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CComponent.php:265
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php(8): CComponent->__call('googlePlaces', Array)
#1 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php(8): CWebApplication->googlePlaces('AIzaSyAiyb9-cE1...')
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require('C:\Users\Arsey\...')
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): CBaseController->renderInternal('C:\Users\Arsey\...', NULL, true)
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): CBaseController->renderFile('C:\Users\Arsey\...', NULL, true)
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): CController->renderPartial('index', NULL, true)
#6 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): CController->render('index')
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('')
#12 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('')
#13 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/
---
2013/01/11 15:58:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:58:41 [error] [php] include(googlePlaces.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\Users\Arsey\Dropbox\sites\YII_1_1_13\YiiBase.php:421)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php(8): spl_autoload_call()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#5 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (8)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php (33)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 15:58:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:59:01 [error] [php] include(googlePlaces.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\Users\Arsey\Dropbox\sites\YII_1_1_13\YiiBase.php:421)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php(8): spl_autoload_call()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#5 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (8)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php (33)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php (13)
2013/01/11 15:59:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:59:20 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/11 15:59:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 15:59:55 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/11 15:59:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:01:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:02:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:02:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:03:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:03:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:04:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/11 16:04:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 09:37:40 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/14 09:37:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 11:48:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\db\CDbConnection.php:369
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CModule.php(387): CDbConnection->init()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CModule.php(103): CModule->getComponent('db')
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(30): CModule->__get('db')
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('')
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('')
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/
---
2013/01/14 11:48:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:57:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:57:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:57:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:57:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:57:42 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/14 13:57:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:58:26 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/14 13:58:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:58:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:59:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:59:03 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/14 13:59:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:59:17 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:184)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (184)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (30)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/14 13:59:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 13:59:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:00:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:02:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:04:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:04:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:05:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:05:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:05:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:07:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:08:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:09:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:09:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:09:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:11:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:11:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:11:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:12:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:17:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:17:54 [error] [php] Undefined property: googlePlaces::$_curlopt_ssl_verifypeer (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:207)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php(16): googlePlaces->search()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#4 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#5 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (207)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (166)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (29)
2013/01/14 14:17:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:18:10 [error] [php] Undefined index: status (C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php:169)
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(126): require()
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CBaseController.php(95): SiteController->renderInternal()
#2 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(869): SiteController->renderFile()
#3 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(782): SiteController->renderPartial()
#4 C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\controllers\SiteController.php(33): SiteController->render()
#5 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(286): SiteController->runAction()
#8 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CController.php(265): SiteController->runActionWithFilters()
#9 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): SiteController->run()
#10 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (169)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\extensions\googlePlaces.php (29)
in C:\Users\Arsey\Dropbox\sites\planteaters\api\protected\views\site\index.php (16)
2013/01/14 14:18:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:19:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 14:19:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/14 15:03:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\Users\Arsey\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\Arsey\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
